packages/new-polymath-issuer/src/pages/DividendsDetails/Presenter.tsx

```diff
@@ -0,0 +1,350 @@
+import React, { Fragment } from 'react';
+import {
+  Button,
+  ButtonFluid,
+  Icon,
+  IconOutlined,
+  IconCircled,
+  icons,
+  Heading,
+  GridRow,
+  Card,
+  CardPrimary,
+  Paragraph,
+  Table,
+  Flex,
+  Box,
+  Text,
+  List,
+  Grid,
+} from '@polymathnetwork/new-ui';
+import { types } from '@polymathnetwork/new-shared';
+
+export interface Props {
+  dividends: types.DividendPojo[];
+  symbol: string;
+}
+
+const columns = [
+  {
+    Header: 'Investor Wallet Address',
+    accessor: 'investorWalletAddress',
+  },
+  {
+    Header: 'Dividends Pre-Tax',
+    accessor: 'dividendsPreTax',
+  },
+  {
+    Header: 'Taxes Withheld (%)',
+    accessor: 'taxesWithheldPercent',
+  },
+  {
+    Header: 'Taxes Withheld (TOKEN)',
+    accessor: 'taxesWithheldTokens',
+  },
+  {
+    Header: 'Dividends Paid',
+    accessor: 'dividendsPaid',
+  },
+  {
+    Header: 'Status of Payment',
+    accessor: 'statusOfPayment',
+  },
+];
+
+const dataSet = [
+  {
+    investorWalletAddress: '0x9F510…00A3f1b',
+    dividendsPreTax: '100,000.00 TOKEN',
+    taxesWithheldPercent: '18%',
+    taxesWithheldTokens: '300.00 TOKEN',
+    dividendsPaid: '100,000.00 TOKEN',
+    statusOfPayment: 'Completed',
+  },
+  {
+    investorWalletAddress: '0x9F510…00A3f1b',
+    dividendsPreTax: '100,000.00 TOKEN',
+    taxesWithheldPercent: '18%',
+    taxesWithheldTokens: '300.00 TOKEN',
+    dividendsPaid: '100,000.00 TOKEN',
+    statusOfPayment: 'Completed',
+  },
+];
+
+export const Presenter = ({ symbol, dividends }: Props) => (
+  <div>
+    <Button variant="ghost" iconPosition="right">
+      Go back
+      <Icon Asset={icons.SvgArrow} width={18} height={17} />
+    </Button>
+    <Heading variant="h1" as="h1">
+      Stunning Dividend Distribution Title Example
+    </Heading>
+    <GridRow>
+      <GridRow.Col gridSpan={{ sm: 12, lg: 4 }}>
+        <Card p="gridGap" height={425}>
+          <Grid height="100%" gridAutoRows="1fr auto">
+            <Box>
+              <Grid
+                mb={4}
+                gridAutoFlow="column"
+                gridGap={0}
+                gridAutoColumns="auto 1fr"
+              >
+                <IconOutlined
+                  Asset={icons.SvgCheckmark}
+                  width={64}
+                  height={64}
+                  color="success"
+                  scale={0.9}
+                  borderWidth="5px"
+                />
+                <Box ml="m">
+                  <Text color="highlightText" fontSize={6} fontWeight={0}>
+                    100%
+                  </Text>
+                  <Paragraph>transactions are completed</Paragraph>
+                </Box>
+              </Grid>
+              <Paragraph>
+                Dividends were successfully distributed to all applicable
+                Investors.
+              </Paragraph>
+            </Box>
+            <Box>
+              <hr />
```

Comment:
We might want to create an actual component for this (very simple one I suppose) so we can have it look exactly how we want.